frontend/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import { Button, Hint, HintBody, HintTitle, PageSection } from '@patternfly/react-core';
+import { TimesIcon } from '@patternfly/react-icons';
+import { useNavigate } from 'react-router-dom';
+import { useLocalStorage } from '@patternfly/quickstarts';
+import ProjectsSection from '~/pages/home/ProjectsSection';
+import OrganizeSection from '~/pages/home/OrganizeSection';
+import QuickStartsSection from '~/pages/home/QuickStartsSection';
+import EnableTeamSection from '~/pages/home/EnableTeamSection';
+import { AccessReviewResourceAttributes } from '~/k8sTypes';
+import { useAccessReview } from '~/api';
+
+const accessReviewResource: AccessReviewResourceAttributes = {
+  group: 'project.openshift.io',
+  resource: 'projectrequests',
+  verb: 'create',
+};
+
+const Home: React.FC = () => {
+  const navigate = useNavigate();
+  const [hintHidden, setHintHidden] = useLocalStorage('rhodsNewLandingPageMessage', null);
+  const [allowCreate, rbacLoaded] = useAccessReview(accessReviewResource);
+
+  return (
+    <>
+      {!hintHidden ? (
+        <PageSection>
+          <Hint
+            actions={
+              <Button
+                aria-label="close welcome info"
+                isInline
+                variant="plain"
+                onClick={() => setHintHidden('true')}
+              >
+                <TimesIcon />
+              </Button>
+            }
+          >
+            <HintTitle>Welcome to your new home page</HintTitle>
+            <HintBody>
+              Your home page summarizes the projects you have access to, and includes helpful
+              resources such as quick starts.
+              <br />
+              The old{' '}
+              <Button
+                style={{ fontSize: 'initial' }}
```

Comment:
PF inline link buttons set the font size smaller. This overrides the setting.

Reply:
Could you log a bug against PF if they cannot provide inline styles like that?

This seems like one of those "override it because it's wrong" and not "we want to do it because we want styles to be prescriptive" -- which has been the DS Projects & Home page goal here.

Reply:
I would like to see the bug logged as a comment above weird edge-case code so we know when we can remove it